src/index.js

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import App from './components/App';
 import reportWebVitals from './reportWebVitals';
+import {Provider} from 'react-redux';
+import {createStore} from 'redux';
+import rootReducer from './reducers';
 
```

Comment:
Kindly adds example books to the initial state object in this file. Since this is the entry point of our application, it would be a great idea to have the initial state object here. Kindly address it